misc/python/materialize/data_ingest/workload.py

```diff
@@ -305,6 +305,7 @@ def execute_workload(
         with conn.cursor() as cur:
             try:
                 cur.execute("SET REAL_TIME_RECENCY TO TRUE")
+                cur.execute("SET real_time_recency_timeout = '60s'")
```

Comment:
Can we leave a note here mentioning the PR that should fix it? (I think we have a system that notifies us when an issue number mentioned in the code closes, right?)